src/src/rx-serial/SerialHoTT_TLM.cpp

```diff
@@ -110,6 +110,7 @@ void SerialHoTT_TLM::pollDevice(uint8_t id)
 {
     // send data request to device
     _outputPort->write(START_OF_CMD_B);
+    delay(1);
```

Comment:
We definitely do not want to be putting delays in the code here.

Reply:
I feared you'd say that and I don't like it either. A few sensors are have slow PICs with softserial and can't handle serial data sent back to back. This part of the software runs worst case about every 60ms. Any idea how to solve this without the delay? If not I'll scrap this PR and well, loose a sensor that would otherwise work.